pkgs/servers/sql/postgresql/ext/pg-semver.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
```

Comment:
Since we're doing another iteration anyways: can you remove the `stdenv` here, it's unused.

Reply:
Thanks, I've removed both.